packages/fxa-content-server/package.json

```diff
@@ -132,8 +132,6 @@
     "photon-colors": "1.0.3",
     "postcss-cli": "7.1.1",
     "postcss-loader": "4.2.0",
-    "react": "^16.8.6",
```

Comment:
[Context why this was in content-server](https://github.com/mozilla/fxa/pull/15491#issuecomment-1615225800)